src/evaluation_system/tests/history_command_test.py

```diff
@@ -52,6 +54,20 @@ def test_freva_history_method(dummy_history, dummy_user):
         hist = history(entry_ids="bla")
     hist = history(entry_ids="0")
     assert len(hist) == 0
+    with patch(
```

Comment:
btw `_history.py` is [100%](https://app.codecov.io/gh/FREVA-CLINT/freva/blob/main/src%2Ffreva%2F_history.py) under coverage, and imo I think there is no need to define new test for it.
Then in consequence we can remove `mock` from setup and make the number of dependencies a bit less. 
But if u wanna keep the test, because u need it in both `docs` and `test`, it would be better to copy `mock` to the `test` as well in `setup.py`. or I think keeping it in `test` would be enough, because it originally comes from `test`

Reply:
I added "mock" into the docs setup.py because for some reason during CI it was searching for it, while when running locally did not

regarding taking out this latest part of the test from history_command_test. Sure,it in get_history_test() is enough

I will see in how many places mock is used (besides docs, apparently).